src/platform/user/tests/authentication/utilities.unit.spec.js

```diff
@@ -26,8 +26,14 @@ describe('authentication URL helpers', () => {
   beforeEach(setup);
 
   it('should redirect for signup v1', () => {
-    signup('v1');
```

Comment:
I think we should keep a test for whatever happens when `signup` is called without a provided `queryParam` so we know and can validate default behavior

Reply:
Yeah I can re-add it, the functionality is not built on the API side so those query parameters will not do anything right now.  The default behavior and the `csp` query parameters will still just navigate to ID.me after going through eAuth.